sdks/nodejs/src/agonesSDK.js

```diff
@@ -19,11 +19,30 @@ const services = require('../lib/sdk_grpc_pb');
 
 class AgonesSDK {
 	constructor() {
-		this.client = new services.SDKClient('localhost:59357', grpc.credentials.createInsecure());
+		this.client = new services.SDKClient('localhost:'+this.port, grpc.credentials.createInsecure());
 		this.healthStream = undefined;
 		this.emitters = [];
 	}
 
+	get port() {
+		const defaultPort = '59357';
```

Comment:
The default port is left in for backwards compatibility. If you use a newer SDK (1.1+) with an older version of Agones (1.0), then the agones controller will not set the environment variable and it would be nice if the sidecar didn't crash. 

We can go back through at some point and remove the defaults, but I think for now we need to leave them in.